modules/libcom/src/fdmgr/fdManager.cpp

```diff
@@ -203,9 +345,9 @@ LIBCOM_API void fdManager::process (double delay)
          * of select()
          */
         epicsThreadSleep(minDelay);
-        this->pTimerQueue->process(epicsTime::getCurrent());
+        priv->pTimerQueue->process(epicsTime::getCurrent());
     }
-    this->processInProg = false;
+    priv->processInProg = false;
```

Comment:
As a note.  I looked for, but did not find any new `return` added above, so I think the `processInProg` would be cleared in call cases excepting an exception.  This was the case before, meaning that an exception from a user callback was/is not recoverable.   I don't think any new `throw` are implicitly added, so no objections.